isis/src/lro/apps/lrowaccal/main.cpp

```diff
@@ -206,14 +207,21 @@ void IsisMain () {
         // Astronomical Units (AU)
         QString bspKernel1 = p.MissionData("lro", "/kernels/tspk/moon_pa_de421_1900-2050.bpc", false);
         QString bspKernel2 = p.MissionData("lro", "/kernels/tspk/de421.bsp", false);
+        NaifStatus::CheckErrors();
```

Comment:
Does there exist a naif call that could error that this would catch before the furnsh calls?

Reply:
The furnsh calls are what are failing, due to the file not existing. The naifstatus calls wrap the internal spicelib error handling, so that we get an error thrown instead of the blind continue that was there before.

Reply:
Guess, I failed to mention that. Yes, the error was not there after inserting the status calls, and if I remove the needed files, it issues an error with the name of the missing file.